extensions/amp-google-read-aloud-player/validator-amp-google-read-aloud-player.protoascii

```diff
@@ -0,0 +1,95 @@
+#
+# Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-google-read-aloud-player
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-google-read-aloud-player"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-google-read-aloud-player>
+  html_format: AMP
+  tag_name: "amp-google-read-aloud-player"
+  requires_extension: "amp-google-read-aloud-player"
+  attrs: {
+    name: "data-api-key"
+    mandatory: true
+  }
+  attrs: {
+    name: "data-tracking-ids"
+    mandatory: true;
+    value_regex_casei: "UA-\\d+-\\d+[\,UA-\\d+-\\d+]+";
+  }
+  attrs: {
+    name: "data-voice"
+    mandatory: true;
+  }
+  attrs: {
+    name: "data-url"
+    value_url {
+      protocol: 'https';
+      protocol: 'http';
```

Comment:
It might be undesirable to allow `http`. Since we're loading the iframe using `https`, the browser could throw mixed-content security warnings.